pkg/mcs/scheduling/server/apis/v1/api.go

```diff
@@ -160,6 +167,26 @@ func (s *Service) RegisterOperatorsRouter() {
 	router.GET("/records", getOperatorRecords)
 }
 
+// RegisterRegionsRouter registers the router of the regions handler.
+func (s *Service) RegisterRegionsRouter() {
+	router := s.root.Group("regions")
+	router.GET("/:id/label/:key", getRegionLabelByKey)
+	router.GET("/:id/labels", getRegionLabels)
+	router.POST("/accelerate-schedule", accelerateRegionsScheduleInRange)
+	router.POST("/accelerate-schedule/batch", accelerateRegionsScheduleInRanges)
+	router.POST("/scatter", scatterRegions)
+	router.POST("/split", splitRegions)
+	router.GET("/replicated", checkRegionsReplicated)
+}
+
+// RegisterRegionLabelRouter registers the router of the region label handler.
+func (s *Service) RegisterRegionLabelRouter() {
+	router := s.root.Group("config/region-label")
```

Comment:
Shall we add a sub group in config?

Reply:
yes, I will do it. The code is written before config pr merged.